app/assets/javascripts/services/sse_factory.js

```diff
@@ -0,0 +1,20 @@
+samson.factory('SseFactory', function($location) {
+  'use strict';
+
+  var sse = {
+    connection: null,
+
+    init: function() {
+      this.connection = new EventSource($location.protocol() + '://' + $location.host() + ':' + $location.port() + '/sse');
```

Comment:
`window.location.origin` (if the port was the default it would be an empty string?)